src/go/k8s/apis/redpanda/v1alpha1/cluster_webhook.go

```diff
@@ -640,6 +649,14 @@ func (r *Cluster) validateRedpandaResources(
 	return allErrs
 }
 
+func (r *Cluster) validateLicense(old *Cluster) field.ErrorList {
+	var allErrs field.ErrorList
+	if old.Spec.LicenseRef != nil && r.Spec.LicenseRef == nil {
+		allErrs = append(allErrs, field.Invalid(field.NewPath("spec").Child("licenseRef"), r.Spec.LicenseRef, "licenseRef must be set once configured"))
```

Comment:
nit 'license cannot be unset'

Reply:
I'm following [this](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#validation) convention